resources/views/sections/site-footer.blade.php

```diff
@@ -0,0 +1,9 @@
+<footer class="site-footer -wrapper">
+  <div class="site-footer__widgets">
+    @php(dynamic_sidebar('sidebar-footer'))
+  </div>
+
+  <p class="site-footer__copyrights">
+    To do: Add copyrights text from ACF options
```

Comment:
For todo's please use this convention:
html:
`<!-- TODO: some todo -->`

js:
`// TODO: some todo`

So we can have syntax highlighting while using vs code plugins like Better Comments or create issues directly through ide (I recommend GitHub Pull Requests and Issues for vs code - ratings are poor but it works for me ;))

cc: @coditive-przemek-h